l4ping/l4ping.TESTGEN.ps1

```diff
@@ -0,0 +1,143 @@
+#===============================================
+# Script Input Parameters Enforcement
+#===============================================
+Param(
+    [parameter(Mandatory=$false)] [string] $Config = "Default",
```

Comment:
Parameter validation?

Reply:
Line 9 in PerfTest.ps1 is validating the config value for all the TestGens. Would you like me to add validation here as well?